cmd/loop/main.go

```diff
@@ -136,53 +136,63 @@ func getMaxRoutingFee(amt btcutil.Amount) btcutil.Amount {
 	return swap.CalcFee(amt, maxRoutingFeeBase, maxRoutingFeeRate)
 }
 
-type limits struct {
-	maxSwapRoutingFee   *btcutil.Amount
-	maxPrepayRoutingFee *btcutil.Amount
+type inLimits struct {
+	maxMinerFee btcutil.Amount
+	maxSwapFee  btcutil.Amount
+}
+
+func getInLimits(quote *looprpc.InQuoteResponse) *inLimits {
+	return &inLimits{
+		// Apply a multiplier to the estimated miner fee, to not get
+		// the swap canceled because fees increased in the mean time.
+		maxMinerFee: btcutil.Amount(quote.MinerFee) * 3,
```

Comment:
Why has this changed from *100 to *3 for loopin? Perhaps pull the multiplier out and add an explanatory comment.

Reply:
I think it didn't, as it's just moved. +1 for not having magic numbers

Reply:
It hasn't changed. Previously `getLimits` was only called for loop out.

Reply:
I can add a constant, but if it's name will be `multiplier`, it doesn't add much. Suggestions?

Reply:
Yeah, if this is a simple empirical multiplier then the comment we currently have is probably enough.